### PR TITLE
Add chartbeat analytics to PGL pages

### LINKED ISSUE #7460: Enable Chartbeat on PGL assets
**Is your feature request related to a problem? Please describe.**
Chartbeat is not enabled on PGL assets. We should enable it to maintain consistency between our pages and analytics providers.

**Describe the solution you'd like**
Enable Chartbeat analytics on the PGL pages.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #7460 

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- Add's the chartbeat beacon to the PGL page
- Updates chartbeat utils to support PGL pages

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

Ensure the chartbeat beacon is firing correctly on PGL assets

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/ChartbeatAnalytics/utils/index.test.js [hunk 1]
```diff
@@ -357,6 +362,19 @@ describe('Chartbeat utilities', () => {
 
       expect(getTitle({ pageType, pageData })).toBe('STY Page Title');
     });
+
+    it('should return correct title when pageType is STY', () => {
```
Comment (jimjohnsonrollings) [inline src/app/containers/ChartbeatAnalytics/utils/index.test.js]: This doesn't match - needs to be `PGL` too :)

File: src/app/pages/PhotoGalleryPage/index.jsx [hunk 2]
```diff
@@ -108,6 +109,7 @@ const PhotoGalleryPage = ({ pageData }) => {
         aboutTags={aboutTags}
       />
       <ATIAnalytics data={pageData} />
+      <ChartbeatAnalytics data={pageData} />
```
Comment (HarveyPeachey) [inline src/app/pages/PhotoGalleryPage/index.jsx]: Just looking at the page tests for STY pages. We seem to be mocking [chartbeat](https://github.com/bbc/simorgh/blob/3eed623efc9ec02764960567aebf6848d572055b/src/app/pages/StoryPage/index.test.jsx#L39), do we want to do the same for the PGL page test?

Comment (chris-hinds) [reply src/app/pages/PhotoGalleryPage/index.jsx]: good idea. Just doing a bit of a reactor on the tests, will include it in that update

**Review by rhenshaw56 — APPROVED** [review]

File: src/app/containers/ChartbeatAnalytics/utils/index.test.js [hunk 3]
```diff
@@ -345,18 +243,67 @@ describe('Chartbeat utilities', () => {
       );
     });
 
-    it('should return correct title when pageType is STY', () => {
-      const pageType = 'STY';
-      const pageData = {
-        promo: {
-          headlines: {
-            headline: 'STY Page Title',
+    test.each`
```
Comment (HarveyPeachey) [inline src/app/containers/ChartbeatAnalytics/utils/index.test.js:246]: New test format looks great!

**Review by HarveyPeachey — APPROVED** [review]

**Pull request merged**